src/modules/organization/mutations.hooks.ts

```diff
@@ -45,3 +51,28 @@ export function useInternalGroupPositionMembershipMutations() {
     quitKSGLoading,
   }
 }
+
+export function useInternalGroupUserHighlightMutations() {
+  const [
+    patchInternalGroupUserHighlight,
+    { loading: patchInternalGroupUserHighlightLoading },
+  ] = useMutation<
+    PatchInternalGroupUserHighlightReturns,
+    PatchInternalGroupUserHighlightVariables
+  >(PATCH_INTERNAL_GROUP_USER_HIGHLIGHT_MUTATION)
+
+  const [
+    batchPatchInternalGroupUserHighlight,
+    { loading: batchPatchInternalGroupUserHighlightLoading },
+  ] = useMutation<
+    BatchPatchInternalGroupUserHighlightReturns,
+    BatchPatchInternalGroupUserHighlightVariables
+  >(BATCH_PATCH_INTERNAL_GROUP_USER_HIGHLIGHT_MUTATION)
```

Comment:
Should we get rid of batch patches? Or are we planning to use them in the future?